### DISCUSSION TITLE
Apache 反向代理，控制台部分链接根目录重复

### DISCUSSION THREAD

**Discussion by UM-Li** [walinejs/waline#1800] [Q&A] [answered]
您好，我将 Waline 端口反代到网站的 `/waline` 目录下：

```apache
<VirtualHost *.443>
  ...
  ProxyPass /waline http://localhost:8360
  ProxyPassReverse /waline http://localhost:8360
</VirtualHost>
```

控制台 UI 的部分链接出现重复的根目录，变成无效路径：
- 评论：`/waline/ui` → `/waline/waline/ui`
- 用户：`/waline/ui/user` → `/waline/waline/ui/user`
- 导入导出：`/waline/ui/migration` → `/waline/waline/ui/migration`
- 用户资料：`/waline/ui/profile` → `/waline/waline/ui/profile`

但其它一些链接则无此问题，如：
- 用户资料页的外部账号绑定：均为 `/waline/oauth/...`
- 登录页各选项：均为 `/waline/ui/...`

`WALINE_ADMIN_MODULE_ASSET_URL` 是默认值，从 unpkg.com 加载。

请问怎么去除这些链接的重复部分？

**Comment by lizheming** [comment, marked as answer]
顶部跳转的路由在子路径适配上有点问题，可以自行拼接地址后跳转，如介意可以先用子域名 https://github.com/orgs/walinejs/discussions/1779

**Reply by UM-Li** [reply]
感谢解答！
